src/engine/vlcengine.cpp

```diff
@@ -98,9 +98,12 @@ bool VLCEngine::Init() {
 
 }
 
-bool VLCEngine::Load(const QUrl &media_url, const QUrl &stream_url, const EngineBase::TrackChangeFlags change, const bool force_stop_at_end, const quint64 beginning_nanosec, const qint64 end_nanosec) {
+bool VLCEngine::Load(const QUrl &media_url, const QUrl &stream_url, const std::optional<double> ebur128_loudness_LUFS, const EngineBase::TrackChangeFlags change, const bool force_stop_at_end, const quint64 beginning_nanosec, const qint64 end_nanosec) {
+
+  // FIXME: why is this not calling `EngineBase::Load()`?
```

Comment:
I think this is because it's not using pre-loading or fading, but you can add it if it's needed.

Reply:
I'd rather not touch that for now...